github/client.go

```diff
@@ -428,6 +428,28 @@ func (client *Client) CreateIssue(project *Project, title, body string, labels [
 	return
 }
 
+func (client *Client) UpdateIssue(project *Project, number int, assignee string) (err error) {
```

Comment:
The `number` variable could be named a little less ambiguously, i.e. `issueNumber`